### PR TITLE
Stepper component changes 17/1/2017

### PR DESCRIPTION
**Pull request by Kani1996**

### CODE REVIEW

**Comment by Kani1996** [comment]
[stepperChangesScreens.docx](https://github.com/karthikkumar1996/angular/files/1639697/stepperChangesScreens.docx)

**Comment by Kani1996** [comment]
Please find the attached screenshots showing the changes for stepper component in Material and theme changes.

**Comment by KarthikKumarDev** [comment]
In brown theme change yellow colour.

**Comment by KarthikKumarDev** [comment]
Next time upload screenshots in this way.....

45388472-hd-blue-wallpaper

File: src/app/app.component.scss [hunk 1]
```diff
@@ -33,13 +35,39 @@
...
+    font-size: 20px;
+    box-shadow: 0px 1px 5px black;
+    background-color : beige;
+  background-repeat: repeat;
+}
+
+
+.page-footer{
+    background-color:beige;
+  background-repeat: repeat;
```
Comment (KarthikKumarDev) [inline src/app/app.component.scss]: Beware of Indentations

File: src/app/app.module.ts [hunk 2]
```diff
@@ -37,6 +42,10 @@ import { GraphComponent } from './graphs/graph.component';
     {
       path: 'graph',
       component: GraphComponent,
+    },
+    {
+      path: 'qualification',
+      component: QualificationComponent,
```
Comment (KarthikKumarDev) [inline src/app/app.module.ts:48]: its a good idea, lets follow with it and have form procuring Qualifications !

**Comment by KarthikKumarDev** [comment]
if you are not making use of the three new assets(image files) remove it from Pull request !

**Comment by KarthikKumarDev** [comment]
Images in the following comment are from stepperChangesScreens.docx shared in older comment by @Kani1996 

image

BAD.  Change Side-Nav to same color as Page Header. might look better !.

**Comment by KarthikKumarDev** [comment]
image

Colors Good ! But we already have a dark theme right ?! maybe add this as third theme.
BUT do format the stepper and other stuff on the page properly. We can work on the fields later, but proper format it.

**Comment by KarthikKumarDev** [comment]
image

Colors Good ! but dude how many dark themes ????! Have the toolbar color to the cards also. Would look good.

File: src/app/qualification/qualification.component.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<h1>Hiiiii</h1>
```
Comment (KarthikKumarDev) [inline src/app/qualification/qualification.component.html]: This is ridiculous. Check for these stuff before commits and pull requests and Remove them

**Comment by KarthikKumarDev** [comment]
Toolbar and side-nav should be same color, or atleast near shade.Let the page header be the old color. Just change the side-nav color to the page header color now. Choose one font color, either black or white.

**Comment by KarthikKumarDev** [comment]
The second theme looks good, but give the stepper element either toolbar or pageheader.(whichever looks good for you !)

**Comment by KarthikKumarDev** [comment]
Excited to see the stepper in function ! Will be deploying soon after these changes. !

**Pull request merged**